Extractor/FailureBuffer.cs

```diff
@@ -216,7 +216,7 @@ public async Task<bool> WriteDatapoints(IEnumerable<UADataPoint> points, IDictio
             if (points == null || !points.Any() || pointRanges == null || !pointRanges.Any() || fullConfig.DryRun) return true;
 
             points = points.GroupBy(pt => pt.Id)
-                .Where(group => !extractor.State.GetNodeState(group.Key)?.FrontfillEnabled ?? false)
+                .Where(group => extractor.State.GetNodeState(group.Key)?.FrontfillEnabled == false)
```

Comment:
What's the change here? Afaik `!null == null` in C#
new condition is only `true` for `false` on the left
old condition is `false` for `null`. `true` for `false` and `false` for `true`, so exactly the same?

Reply:
Yep. It's just a lot clearer IMO.

Reply:
!null is not in fact a thing in C# at all, `??` has higher operator precedence, but it is _not_ obvious.